src/components/App/app.css

```diff
@@ -0,0 +1,66 @@
+.main {
+  display: grid;
+  height: 100vh;
+  grid-template-rows: 50px auto;
+  grid-template-columns: 190px auto;
+  grid-template-areas:
+    "menu header"
+    "menu content";
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  background-color: #fff;
+  align-items: center;
+  justify-content: flex-end;
+  padding-right: 45px;
+  grid-area: header;
+}
+
+.header__name {
+  color: gray;
+  padding-right: 25px;
+}
+
+.header__button {
+  width: 55px;
+  height: 25px;
+  background-color: transparent;
```

Comment:
remove this and set it on the index.css like this

```
button {
 background-color: transparent;
}
```

so in this way you will remove the background to all buttons on the app